concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -172,38 +174,34 @@ def main():
         REQUESTOR_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         headers = {
-            'Content-Type': 'application/octet-stream',
+            'Content-Type'             : 'application/octet-stream',
             'concent-client-public-key': b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
-        }
+        },
+        expected_status = 200
     )
 
+
     #  Test CASE 2B - Send ForceSubtaskResults with not enough amount of funds on account
     api_request(
         cluster_url,
         'send',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         force_subtask_results(
-            timestamp = timestamp_to_isoformat(current_time),  # current_time
-            ack_report_computed_task = ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 14401),  # current_time - 4:00:01
-                subtask_id = task_id,
-                task_to_compute = task_to_compute(
-                    timestamp = timestamp_to_isoformat(current_time - 15401),  # current_time - 4:00:05
-                    compute_task_def = compute_task_def(
-                        task_id     = task_id,
-                        subtask_id  = task_id,
-                        deadline    = current_time + 3600,  # current_time + 01:00:00
-                    )
-                )
+            timestamp = current_timestamp,  # current_time
+            ack_report_computed_task    = ack_report_computed_task(
+                    timestamp           = ack_report_computed_task_timestamp,  # current_time - 4:00:10
+                    subtask_id          = task_id,
```

Comment:
You are adding + '2A' to previous calls, I think it is worth adding '2B' here to for consistency, and for easier examination in logs/database if we will have to debug.

Reply:
done